Sources/Transaction/Memos/SenderWithPaymentRequestMemo.swift

```diff
@@ -24,21 +24,27 @@ struct RecoverableSenderWithPaymentRequestMemo {
     }
 
     func recover(senderPublicAddress: PublicAddress) -> SenderWithPaymentRequestMemo? {
-        guard SenderWithPaymentRequestMemoUtils.isValid(memoData: memoData,
-                                       senderPublicAddress: senderPublicAddress,
-                                       receipientViewPrivateKey: accountKey.subaddressViewPrivateKey,
-                                       txOutPublicKey: txOutPublicKey) else {
+        guard SenderWithPaymentRequestMemoUtils.isValid(
+            memoData: memoData,
+            senderPublicAddress: senderPublicAddress,
+            receipientViewPrivateKey: accountKey.subaddressViewPrivateKey,
+            txOutPublicKey: txOutPublicKey)
```

Comment:
double-indent function call params:
```suggestion
                memoData: memoData,
                senderPublicAddress: senderPublicAddress,
                receipientViewPrivateKey: accountKey.subaddressViewPrivateKey,
                txOutPublicKey: txOutPublicKey)
```

Reply:
single indentation is the general rule